src/ruby/integration/multibuildpack_test.go

```diff
@@ -0,0 +1,92 @@
+package integration_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/cloudfoundry/switchblade"
+	"github.com/sclevine/spec"
+
+	. "github.com/cloudfoundry/switchblade/matchers"
+	. "github.com/onsi/gomega"
+)
+
+func testMultiBuildpack(platform switchblade.Platform, fixtures string) func(*testing.T, spec.G, spec.S) {
+	return func(t *testing.T, context spec.G, it spec.S) {
+		var (
+			Expect     = NewWithT(t).Expect
+			Eventually = NewWithT(t).Eventually
+
+			name string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = switchblade.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(platform.Delete.Execute(name)).To(Succeed())
+		})
+
+		context("when ruby is a supply for the binary buildpack", func() {
+			it("finds the supplied dependency in the runtime container", func() {
+				deploymentProcess := platform.Deploy.
+					WithBuildpacks(
+						"ruby_buildpack",
+						"https://github.com/cloudfoundry/binary-buildpack#master",
```

Comment:
Do we wanna initialize all buildpacks like we do Go in init_test?

Reply:
I copied this out from the existing test. I don't think it matters whether we initialize all buildpacks or not. It's basically the difference between the operator installing buildpacks on the platform vs the user requesting them via `-b <buildpack>` during `cf push`.